jekyll/index.html

```diff
@@ -281,6 +281,7 @@ <h2>Real-time chat</h2>
         <p><a href="https://matrix.to/#/#freenode_#nim:matrix.org"><i class="fab i">m</i>#nim:matrix.org</a></p>
         <p><a href="https://irclogs.nim-lang.org"><i class="fc fa-irc">#</i>IRC Logs</a></p>
         <p><a href="https://t.me/nim_lang"><i class="fab fa-telegram"></i>Telegram/nim_lang</a></p>
+	<p><a href="https://t.me/nimitalia"><i class="fab fa-telegram"></i>Telegram/nim_italia</a></p>
```

Comment:
I don't think we should be adding too much of this to the front page, it'll get noisy. Please remove this one.

Reply:
Agreed. There are also other local groups (Spanish for sure, maybe Russian) which are not mentioned here.

Reply:
Ok, I will remove that